yarn.lock

```diff
@@ -1592,7 +1592,7 @@ create-hmac@^1.1.0, create-hmac@^1.1.2, create-hmac@^1.1.4:
     safe-buffer "^5.0.1"
     sha.js "^2.4.8"
 
-create-react-context@^0.2.1:
+create-react-context@^0.2.3:
```

Comment:
this won't fix the issue - consider to review text of LICENSE for this version, we won't be able to comply. Consider to enforce version 0.2.2 instead (or lower) - and as the newer version will be published (assuming the LICENSE would still be MIT) we will be able to update the version here.

Reply:
I don't understand , 0.2.3 has MIT license..

Reply:
oh... the license was introduced in 0.2.3...then why are we even bothering?

Reply:
`create-react-context@0.2.2`

Reply:
agreed to make changes in package.json instead, to include `resolutions` section: https://yarnpkg.com/lang/en/docs/selective-version-resolutions/

Reply:
done, @kuzhelov can u please take another look?